CONTRIBUTING.md

```diff
@@ -22,6 +23,23 @@ This project has adopted the [Microsoft Open Source Code of Conduct](https://ope
 kubectl scale deployments/inflate --replicas=3
 ```
 
+### Use local environment instead of GitHub Codespaces. 
+> Tested environment : WSL2
```

Comment:
Should specify distribution, especially since package management is distribution-specific